apps/web/components/Gallery/__tests__/Gallery.spec.ts

```diff
@@ -5,16 +5,16 @@ describe('<Gallery />', () => {
   it('should render component', () => {
     const { getByTestId } = mount(Gallery, {
       props: {
-        images: [{ 
-          url: '/images/test.webp', 
-          width: 400, 
+        images: [{
+          url: '/images/test.webp',
+          width: 400,
           height: 573,
           urlMiddle:'',
           urlPreview:'',
           urlSecondPreview: '',
           cleanImageName: '',
           path: '',
-          names: [''],
+          names: [],
```

Comment:
```suggestion
          names: [''],
```

I think this is no longer necessary.